examples/gno.land/p/demo/audio/bytebeat/bytebeat.gno

```diff
@@ -0,0 +1,70 @@
+package bytebeat
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/base64"
+	"math"
+
+	"gno.land/p/demo/audio/biquad"
+	"gno.land/p/demo/audio/wav"
+	"gno.land/p/demo/ufmt"
+)
+
+func ByteBeat(seconds uint32, sampleRate uint32, bytebeat_func func(t int) int) (data string) {
+	b := new(bytes.Buffer)
+	w := bufio.NewWriter(b)
+
+	var numSamples uint32 = sampleRate * seconds / 2
```

Comment:
type unnecessary (sampleRate and seconds are uint32); make it `:=`?

Reply:
Fixed! d26d1261